blocks/editable/format-toolbar/index.js

```diff
@@ -93,6 +99,17 @@ class FormatToolbar extends Component {
 		};
 	}
 
+	toggleLinkSettingsVisibility() {
+		const settingsVisible = ! this.state.settingsVisible;
+		this.setState( { settingsVisible } );
```

Comment:
there's an implicit coding standard stating that if the next state depends on the previous, we should use the callback notation (I think we were trying to add an eslint for this in an old PR)

```js
this.setState( ( state ) => ( { settingsVisible: ! state.settingsVisible } ) );
```

Reply:
Fixed!